near/test/msg.ts

```diff
@@ -0,0 +1,91 @@
+// npx pretty-quick
+
+import Web3 from "web3";
+import { NodeHttpTransport } from "@improbable-eng/grpc-web-node-http-transport";
+
+import { TestLib } from "./testlib";
+
+import { Account as nearAccount } from "@certusone/wormhole-sdk/node_modules/near-api-js";
+const BN = require("bn.js");
+const fetch = require("node-fetch");
+
+const nearAPI = require("near-api-js");
+
+export function textToHexString(name: string): string {
+  return Buffer.from(name, "binary").toString("hex");
```

Comment:
You could use "ascii" instead of "binary" or just not specify anything and it may be more intuitive considering the function name.  But, then again, it is test code.  So....

Reply:
Agreed.. but I don't want to touch this and reset the approvals.. :)